x-pack/plugin/security/qa/security-trial/src/javaRestTest/java/org/elasticsearch/xpack/security/apikey/ApiKeyRestIT.java

```diff
@@ -176,4 +177,24 @@ public void testGrantApiKeyWithoutApiKeyNameWillFail() throws IOException {
         assertEquals(400, e.getResponse().getStatusLine().getStatusCode());
         assertThat(e.getMessage(), containsString("api key name is required"));
     }
+
+    public void testGrantApiKeyWithoutGrantApiKeyPrivilegeFails() throws IOException {
```

Comment:
Adding the test here instead of as a yaml REST test since there currently are none for granting API keys; happy to add some in a separate PR.

Reply:
A seperate PR is fine. It's also not entirely necessary to have both yaml REST test and Java REST test.